### PR TITLE
[Test]Add downlod_extra_resources coverage for lambda e2e

### PR DESCRIPTION
**Pull request by yubozhao**
## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature and improvements (non-breaking change which adds/improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code Refactoring (internal change which is not user facing)
- [ ] Documentation
- [x] Test, CI, or build

## Component(s) if applicable
- [ ] BentoService (service definition, dependency management, API input/output adapters)
- [ ] Model Artifact (model serialization, multi-framework support)
- [ ] Model Server (mico-batching, dockerisation, logging, OpenAPI, instruments)
- [ ] YataiService gRPC server (model registry, cloud deployment automation)
- [ ] YataiService web server (nodejs HTTP server and web UI)
- [ ] Internal (BentoML's own configuration, logging, utility, exception handling)
- [ ] BentoML CLI

## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: e2e_tests/aws_lambda/test_lambda_deployment.py [hunk 1]
```diff
@@ -44,3 +44,13 @@ def test_aws_lambda_deployment(iris_clf_service):
...
+
+
+def test_aws_lambda_deployment(iris_clf_service):
+    run_lambda_deployment(iris_clf_service)
+
+
+def test_aws_lambda_deployment_with_large_file(
+    iris_clf_service_with_additional_large_file
+):
+    run_lambda_deployment(iris_clf_service_with_additional_large_file)
```
Comment (parano) [inline e2e_tests/aws_lambda/test_lambda_deployment.py:56]: could you somehow verify that files were uploaded to s3 and downlod_extra_resources is invoked?

Comment (yubozhao) [reply e2e_tests/aws_lambda/test_lambda_deployment.py:56]: hmmm, I will give it a shot

Comment (parano) [reply e2e_tests/aws_lambda/test_lambda_deployment.py:56]: might be best to just add a large dependency and import the library in the BentoService code and ensure there's no import error

Comment (parano) [reply e2e_tests/aws_lambda/test_lambda_deployment.py:56]: verifying that s3 bucket created etc should be straightforward right?

Comment (yubozhao) [reply e2e_tests/aws_lambda/test_lambda_deployment.py:56]: Regardless of a large file/dependency or not, we will always have an s3 bucket created. Although maybe I can check is the file/dependency uploaded or not

**Pull request closed without merging**